scoresheets/EGPSR.tex

```diff
@@ -2,14 +2,11 @@
 
 \begin{scorelist}
 	\scoreheading{Main Goal}
-	\scoreitem[3]{800}{Executing the task associated with each command}
+	\scoreitem[3]{800}{Find and solve a problem}
```

Comment:
I would like to split this up into 

\scoreitem[3]{250}{Find and communicate a problem}
\scoreitem[3]{550}{Solve a problem}

This would increase the partial scoring and allow the viewers to better understand the reasoning of the robot + generally what is going on. Reasoning for 250: 200 for the location + 50 for the problem class

Reply:
250 seems a lot for just finding. I'll change it to 150/650.